docs/audio/Samplebank_XML.md

```diff
@@ -0,0 +1,77 @@
+# Samplebank XML Format Specification
+
+Samplebank XMLs describe a samplebank file that contains compressed waveform data. It specifies which sample files to include as well as certain global properties such as the index of this samplebank.
+
+---
+
+```xml
+<SampleBank
+    Name="<C Indentifier>"
+    Index="<uint>"
+    Medium="<Medium>"
+    CachePolicy="<CachePolicy>"
+    BufferBug="[bool]"
+>
+```
+Begins a new samplebank.
+
+**Properties**
```

Comment:
technically in XML parlance these are called "attributes", and what's called "Sub-Attributes" below are "tags" (maybe "child tags" or "children"?). I think using the word "properties" is probably fine, but using the word "attribute" for tags could be confusing

(ditto with other XML docs)